extras/combine_jsons.py

```diff
@@ -0,0 +1,38 @@
+import json
+import numpy as np
+
+def join_json_files(list_of_jsons, nametag="model"):
```

Comment:
Please use `argparse` to enable running this from the command line. Right now, I will have to open a python session to run this, which should not be required to run this. The way to do this is as follows. Have the current function `join_json_files` the way it currently is, but at the bottom, add these lines:
```
if "__name__" = "__main__":
    import argparse

    parser = argparse.ArgumentParser()
    parser.add_argument("-C", "--cache", action="store",
                                          help="A cache of JSON file names that needs to be joined")
    parser.add_argument("-t", "--tag", action="store", help="name-tag of the output JSON file")

    args = parser.parse_args()

    # Write here lines of logic that will first read the cache file and then create the list_of_jsons
    # ... 
    # ...
    join_json_files(list_of_jsons, nametag=args.tag)

```